src/TestSetExtensions.jl

```diff
@@ -57,6 +57,17 @@ function ExtendedTestSet(desc; wrap=DefaultTestSet)
     ExtendedTestSet{wrap}(desc)
 end
 
+function isVector(e)
+    if e.head === :vect
+        return true
+    end
+    #Float32 or Int32 arrays get here as Ref's to Vector
```

Comment:
And this is not quite true. The arrays get here with the same _syntax_ as a `Ref`, but they're not actually a ref to a vector. Just a bit of a confusing overlap in the display of vectors and the syntax for `Ref`s.